Makefile

```diff
@@ -107,8 +113,12 @@ dist: clean ## builds source and wheel package
 release: dist ## package and upload a release
 	python -m flit publish dist/*
 
-install: clean ## install the package to the active Python's site-packages
+ESMF_VERSION := $(shell cat $(ESMFMKFILE) | grep "ESMF_VERSION_STRING=" | awk -F= '{print $$2}' | tr -d "'")
+install-esmpy: clean ## install esmpy from git based on installed ESMF_VERSION
+	pip install git+https://github.com/esmf-org/esmf.git@v$(ESMF_VERSION)\#subdirectory=src/addon/esmpy
+
+install: install-esmpy ## install the package to the active Python's site-packages
```

Comment:
@Zeitsperre We can leave this here as an additional security, but this should not be required anymore (yay!)

- If `xESMF` is badly installed or initiated, `xscen` will simply deactivate the functions using it. `xhydro/xdatasets` do not use it at all, for now.
- `esmf/esmpy 8.6.0` has been released and should have fixed that bug too.

Reply:
Let's leave that for another PR.

Reply:
Just wanted to point out that `xdatasets` has `xesmf` as a dependency because of `xagg`. I had initially created a spinoff `xagg` library with no `xesmf` dependency (because `xesmf` is used for regridding but not for spatial averaging in `xagg`, which is the only thing required for `xdatasets`) but we now have `xagg` as a dependency.

`xesmf` is required for the notebooks but might not be for the tests.